roles/database/tasks/initialize-raw-schema.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: List all schemas in the fpsd database.
+  become: true
+  become_user: postgres
+  command: psql -c '\dn' fpsd
```

Comment:
This is a read-only command, meaning nothing changes on the target system, so set `changed_when: false` so it doesn't falsely report "changed". Same for two other read-only tasks listing db info in the database role.